pkg/check/pss/pss.go

```diff
@@ -9,37 +9,64 @@ import (
 	"time"
 
 	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/beekeeper"
 	"github.com/ethersphere/beekeeper/pkg/random"
 	"github.com/gorilla/websocket"
 	"github.com/prometheus/client_golang/prometheus/push"
 )
 
-// Options represents PSS check options
+// Options represents check options
 type Options struct {
-	NodeCount      int
-	RequestTimeout time.Duration
 	AddressPrefix  int
-	Seed           int64
+	MetricsPusher  *push.Pusher
+	NodeCount      int
 	PostageAmount  int64
-	PostageWait    time.Duration
 	PostageDepth   uint64
+	PostageWait    time.Duration
+	RequestTimeout time.Duration
+	Seed           int64
 }
 
-var (
-	errDataMismatch        = errors.New("pss: data sent and received are not equal")
-	errWebsocketConnection = errors.New("pss: websocket connection terminated with an error")
-)
+// NewDefaultOptions returns new default options
+func NewDefaultOptions() Options {
+	return Options{
+		AddressPrefix:  1,
+		MetricsPusher:  nil,
+		NodeCount:      1,
+		PostageAmount:  1,
+		PostageDepth:   16,
+		PostageWait:    5 * time.Second,
+		RequestTimeout: 5 * time.Minute,
+		Seed:           random.Int64(),
+	}
+}
+
+// compile check whether Check implements interface
+var _ beekeeper.Action = (*Check)(nil)
```

Comment:
are these necessary?

Reply:
it is used to confirm that check satisfies the interface, won't compile if it doesn't

Reply:
hmm, it compiles on my machine 🤔